### PR TITLE
Prepare what's new for 6.2 release.

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: docs/source/whatsnew/version6.rst [hunk 1]
```diff
@@ -2,6 +2,55 @@
...
 
+
+.. _whatsnew620:
+
+IPython 6.2
+===========
+
+
+IPython 6.2 contains all the bugs fixes and features :ref:`available in IPython 5.5 <whatsnew550>`,
+like built in progress bar support, and system wideconfiguration
```
Comment (ian-r-rose) [inline docs/source/whatsnew/version6.rst]: system wideconfiguration -> system-wide configuration

File: docs/source/whatsnew/version6.rst [hunk 2]
```diff
@@ -2,6 +2,55 @@
...
+
+The following features are specific to IPython 6.2:
+
+Function signature in completions
+---------------------------------
+
+Terminal IPython will now show the signature of the function while completing.
+Only the currently highlighted function will show its signature on the line
+below the completer by default. The functionality is recent so might be
```
Comment (ian-r-rose) [inline docs/source/whatsnew/version6.rst]: "this functionality is recent, so it might be limited"

File: docs/source/whatsnew/version6.rst [hunk 3]
```diff
@@ -2,6 +2,55 @@
...
+limited, we welcome bug report and enhancement request on it. :ghpull:`10507`
+
+Assignments return values
+-------------------------
+
+IPython can now trigger the display hook on last assignment of cells.
```
Comment (ian-r-rose) [inline docs/source/whatsnew/version6.rst]: "the last assignment"

File: docs/source/whatsnew/version6.rst [hunk 4]
```diff
@@ -2,6 +2,55 @@
...
+
+The following features are specific to IPython 6.2:
+
+Function signature in completions
+---------------------------------
+
+Terminal IPython will now show the signature of the function while completing.
+Only the currently highlighted function will show its signature on the line
+below the completer by default. The functionality is recent so might be
+limited, we welcome bug report and enhancement request on it. :ghpull:`10507`
```
Comment (ian-r-rose) [inline docs/source/whatsnew/version6.rst]: "...limited; we welcome bug reports and requests for enhancements"

**Review by ian-r-rose — APPROVED** [review]

**Comment by Carreau** [comment]
Fixed, Thanks

**Comment by Carreau** [comment]
Thanks !

**Pull request merged**